diff -r -u --from-file=denise-5.1.1 denise-5.1.1/src/DeniseGUI.c denise-5.1.1-fixed/src/DeniseGUI.c
--- denise-5.1.1/src/DeniseGUI.c	2003-04-15 11:41:40.000000000 +0200
+++ denise-5.1.1-fixed/src/DeniseGUI.c	2005-03-02 06:30:09.644451144 +0100
@@ -95,8 +95,7 @@
 		system(command);
 		if(command) free(command);
 	} else closedir(denisedir);
-	free(denisedir);
-	
+	// free(denisedir);
 
 	if ((apphandle=(struct DeniseAppHandle*)UNN_AllocNode(sizeof(struct DeniseAppHandle))))
 	{
